src/app/zap-templates/zcl/data-model/chip/global-structs.xml

```diff
@@ -27,10 +27,20 @@ Git: 0.7-summer-2025-511-g2b8c413df
     <item fieldId="1" name="StatusCode" type="status"/>
   </struct>
 
+  <struct name="AttributionData" apiMaturity="provisional" isFabricScoped="true">
```

Comment:
Is attribution actually in 1.5?  If it's not, please don't add it here, and figure out why Alchemy decided to do so.